great_expectations/rule_based_profiler/types/data_assistant_result/data_assistant_result.py

```diff
@@ -46,6 +45,12 @@ class DataAssistantResult(SerializableDictDot):
     Use "batch_id_to_batch_identifier_display_name_map" to translate "batch_id" values to display ("friendly") names.
     """
 
+    # A mapping is defined for which metrics to plot and their associated expectations
+    EXPECTATION_METRIC_MAP = {
+        "expect_table_row_count_to_be_between": "table.row_count",
+        "expect_column_unique_value_count_to_be_between": "column.distinct_values.count",
+    }
```

Comment:
Any way we can map types? So Expectations to Metrics (as opposed to their string equivalents)? Maybe not but figure its worth a shot so we can avoid magic strings.

Reply:
@NathanFarmer Let us also add a type hint for this constant.  Thanks.

Reply:
@cdkini I think that if @NathanFarmer is mapping expectation types to the associated metrics, it may be acceptable in this case; otherwise, how would he be able to group the different entities for display?

Reply:
@cdkini - Alex is correct. We need a way to map expectations to their associated metrics and if new expectations are added to a custom `DataAssistant` then how will the new `DataAssistantResult` know how to do the mapping?

Reply:
@alexsherstinsky adding a type hint will throw `Mutable default ... is not allowed` error due to this being a dataclass. I suspect this is why you did not type hint the `ALLOWED_KEYS` set directly below this constant.

Reply:
@NathanFarmer Sorry -- you are correct -- please ignore my comment.  Thank you (dataclasses are not easy to work with).